src/dat/sys/routes.clj

```diff
@@ -0,0 +1,44 @@
+(ns dat.sys.routes
```

Comment:
Also right now this only handles GET. Should maybe make it more generic or configurable.